pkg/monitoring/pb/data-feeds/on-chain/registry/feed_updated.proto

```diff
@@ -10,19 +10,20 @@ message FeedUpdated {
   string feed_id = 1; // bytes as hex string for readability
   uint32 observations_timestamp = 2;
   bytes benchmark = 3;
-  bytes report = 4;
+  bytes bundle = 4;
```

Comment:
Why do we need to introduce this field? Is not it a breaking change for the beholder?